src/solver.cpp

```diff
@@ -700,6 +701,7 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
              Primitive::MultiMarginLoss,
              multimarginloss::MultiMarginLossForward{}.SolverDbId());
 
```

Comment:
Why adding a line after the previous op?

Reply:
I follow previous ops (of course some add a line, some don't so I guess is it optional to add a line here)